main.go

```diff
@@ -268,10 +272,13 @@ func generate(in configInput) ([]byte, error) {
 		// Authorities with an empty server config will end up using
 		// the top-level server config. For more details, see:
 		// https://github.com/grpc/proposal/blob/master/A47-xds-federation.md#bootstrap-config-changes.
-		c.Authorities = map[string]struct{}{
+		c.Authorities = map[string]Authority{
 			tdURI: {},
 			"":    {},
 		}
+		if in.includeC2PAuthority {
+			c.Authorities[c2pAuthority] = Authority{}
```

Comment:
We don't want this to be empty, right. If it is empty, this authority will also use the top-level server config which will send the xDS requests to `trafficdirector.googleapis.com:443`. We want requests matching this authority to go to `dns:///directpath-pa.googleapis.com` instead.

Reply:
oh right and use DP. I think I was just talking to @apolycn about it now